src/transac/components/top-bar/filters-selector/filters-selector.html

```diff
@@ -0,0 +1,12 @@
+<div class="filters-selector">
+  <div class="btn-group" uib-dropdown on-toggle="$ctrl.toggleSelector(open)" auto-close="outsideClick">
+    <button id="dropdown-toggle-btn" type="button" class="btn btn-primary" ng-class="{ 'disabled': $ctrl.isDisabled }" ng-click="$ctrl.toggleSelector()" uib-dropdown-toggle>
```

Comment:
Not sure if it was working because of `uib-dropdown-toggle` - I will take a look, thanks